codespell_lib/data/dictionary.txt

```diff
@@ -25994,6 +25995,8 @@ specifiction->specification
 specifictions->specifications
 specificy->specify, specificity, specifically,
 specifid->specified
+specifig->specific
```

Comment:
Suggest specifying too maybe or do we feel that's too much of a stretch?

Reply:
I looks a bit like a stretch. But I don't mind adding it, if you would like to have it in.

Reply:
I guess it's a bit phonetic too maybe?

Reply:
I think it's too much of a stretch too — let's just leave it as-is for right now. wdyt @peternewman?